packages/context/test/unit/inject.unit.ts

```diff
@@ -12,10 +12,10 @@ import {
 
 describe('function argument injection', () => {
   it('can decorate class constructor arguments', () => {
-    // tslint:disable-next-line:no-unused-variable
     class TestClass {
-      constructor(@inject('foo') foo: string) {}
+      constructor(@inject('foo') public foo: string) {}
     }
+    disableUnusedLocalError(TestClass);
```

Comment:
Sigh, this is ugly but it does not seem that TS has fine control or comment based rules.